source/js/components/join/join.jsx

```diff
@@ -64,23 +51,34 @@ export default class JoinUs extends React.Component {
       'has-danger': !this.state.signupSuccess && this.state.userSubmitted && !this.refs.privacy.checked
     });
 
+    let signupState = classNames({
+      'row': true,
+      'py-3': true,
+      'signup-success': this.state.signupSuccess && this.state.userSubmitted,
+      'signup-fail': !this.state.signupSuccess && this.state.userSubmitted
+    });
+
     return (
-      <div hidden={this.state.isHidden} className="row">
-        <div className="col-12">
-          <div className="row d-flex justify-content-end">
-            <button className="close-button" onClick={this.hide}>X</button>
-          </div>
-        </div>
-        <div className="col-12 hidden-md-up text-center mb-2 mt-4">
-          <img width="33%" src={`/_images/burst${this.state.signupSuccess ? `2` : `1`}.svg`}/>
-        </div>
-        <div className="col-12 my-3">
-          <h1 className="h1-white">{!this.state.signupSuccess ? `Stay Connected` : `Thank You`}</h1>
+      <div className={signupState}>
+        <div className="col text-center mb-2 join-graphic">
+          <img src={`/_images/burst${this.state.signupSuccess ? `2` : `1`}.svg`}/>
         </div>
-        <div className="col-md-6 mb-3">
-          {!this.state.signupSuccess ?
-          <div className="py-md-4">
-            <p className="body-black">Sign up for opportunities and news related to a healthy internet.</p>
+        <div className="col">
+          <div className="row">
+            <div className="col-12 join-content">
+              <div className="my-3 join-page-title">
+                <h2 className="h1-white">{!this.state.signupSuccess ? `${this.props.ctaHeader}` : `Thank You`}</h2>
+              </div>
+              <div className="join-heading">
+                <h2 className="h3-black">{!this.state.signupSuccess ? `${this.props.ctaHeader}` : `Thank You`}</h2>
+              </div>
+              {!this.state.signupSuccess ?
+                <p className="body-black" dangerouslySetInnerHTML={{__html:this.props.ctaDescription}}></p>
+                : <p>{this.props.thankYouMessage}</p>
+              }
+            </div>
+          { !this.state.signupSuccess ?
```

Comment:
I find these massive ternary blocks difficult to read/parse. Can you use [this method](https://facebook.github.io/react/docs/conditional-rendering.html#inline-if-with-logical--operator) instead?